local-cli/server/runServer.js

```diff
@@ -83,7 +83,7 @@ function getPackagerServer(args, config) {
       'bmp', 'gif', 'jpg', 'jpeg', 'png', 'psd', 'svg', 'webp', // Image formats
       'm4v', 'mov', 'mp4', 'mpeg', 'mpg', 'webm', // Video formats
       'aac', 'aiff', 'caf', 'm4a', 'mp3', 'wav', // Audio formats
-      'html', // Document formats
+      'html', 'pdf', // Document formats
```

Comment:
Could we move this config to https://github.com/facebook/react-native/blob/master/packager/rn-cli.config.js? It would allow everyone to add the extensions they need.